### PR TITLE
Add renderer repository.

### PR DESCRIPTION
**Pull request by oleksii-demedetskyi**

### CODE REVIEW

File: VideoRenderer/VideoRenderer/RendererRepository.swift [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    
+    public enum Event {
+        case rateChanged(Float)
+        case urlChanged(URL?)
+        case itemStatusChanged(AVPlayerItemStatus)
+        case durationChanged(TimeInterval?)
+        case playbackFinished
+        case bufferFullChanged(Bool)
+        case playbackWillKeepUpChanged(Bool)
+        case loadedTimeRangedChanged([CMTimeRange])
```
Comment (AndriiMoskvin) [inline VideoRenderer/VideoRenderer/RendererRepository.swift]: Have you missed`currentTime` changes?

Comment (oleksii-demedetskyi) [reply VideoRenderer/VideoRenderer/RendererRepository.swift]: Yep.

File: VideoRenderer/VideoRenderer/RendererRepository.swift [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+            self.rate = rate
+            self.volume = volume
+            self.newTime = newTime
+        }
+    }
+    
+    public enum Event {
+        case rateChanged(Float)
+        case urlChanged(URL?)
+        case itemStatusChanged(AVPlayerItemStatus)
```
Comment (AndriiMoskvin) [inline VideoRenderer/VideoRenderer/RendererRepository.swift]: What if the renderer is not connected with `AVPlayer`?

Comment (oleksii-demedetskyi) [reply VideoRenderer/VideoRenderer/RendererRepository.swift]: `rateChanged` -> `playbackStarted` `playbackStopped`
`itemStatusChanged` -> `playbackReady` `playbackFailed(Error)`
`durationChanged` -> `durationReceived(TimeInteval)`

Comment (oleksii-demedetskyi) [reply VideoRenderer/VideoRenderer/RendererRepository.swift]: durartion -> `CMTime`
+ `currentTimeUpdated(CMTime)`

**Review by AndriiMoskvin — APPROVED** [review]

**Pull request merged**